arangosh/Import/ImportHelper.cpp

```diff
@@ -656,6 +716,7 @@ void ImportHelper::ProcessCsvAdd(TRI_csv_parser_t* parser, char const* field, si
 
 void ImportHelper::addField(char const* field, size_t fieldLength, size_t row,
                             size_t column, bool escaped) {
+  std::string lookUpTablevalue(field, fieldLength);
```

Comment:
Allocating this string at the start of the method will cause overhead even if the feature is not used. It would be ideal to not cause measurable overhead in case the feature is not used.